folderprint/README.md

```diff
@@ -0,0 +1,45 @@
+# Go training 
+Repository for the training of Go language.
+
+## Description
+
+The project contains:
+- list all files and folders in a given path
+- cache it for 5 minutes using redis
+- docker-compose.yml file for redis server
+
+## Getting Started
+
+### Language
+
+Git
+
+Go
+
+### Execution
+
+Continued commands from main readme.md
+
+- Run the go mod tidy command to add any missing dependencies to the go.mod file
+```
+go mod tidy
```

Comment:
You don't need to run `go mod init ...`. And `go.mod|sum` files should be part of the project.